L1Trigger/DemonstratorTools/interface/codecs/etsums.h

```diff
@@ -0,0 +1,27 @@
+
+#ifndef L1Trigger_DemonstratorTools_codecs_EtSum_h
+#define L1Trigger_DemonstratorTools_codecs_EtSum_h
+
+#include <array>
+#include <vector>
+
+#include "ap_int.h"
+
+#include "DataFormats/Common/interface/View.h"
+#include "DataFormats/L1Trigger/interface/EtSum.h"
+#include "DataFormats/Math/interface/LorentzVector.h"
```

Comment:
I think that a few of these included header should be either moved to the implementation file (e.g. `LorentzVector.h`) or removed because not needed (if I am not wrong, `BoardData.h` and `L1TkEtMissEmuAlgo.h`)

The same applies to the other files in iterface/codecs

Could you please have a look?

Reply:
Done (L1TkEtMissEmuAlgo.h is needed, BoardData.h is not)